apps/meteor/app/lib/server/methods/sendMessage.ts

```diff
@@ -135,9 +136,9 @@ Meteor.methods<ServerMethods>({
 		}
 
 		try {
-			return await executeSendMessage(uid, message, previewUrls);
+			return await applyAirGappedRestrictionsValidation(() => executeSendMessage(uid, message, previewUrls));
 		} catch (error: any) {
-			if ((error.error || error.message) === 'error-not-allowed') {
+			if ((error.error || error.message) === 'error-not-allowed' || (error.error || error.message) === 'restricted-workspace') {
```

Comment:
Should we change this for something like an includes?
```suggestion
			if (['error-not-allowed', 'restricted-workspace'].includes(error.error || error.message)) {
```